stories/ShouldHandleZoom.js

```diff
@@ -28,8 +28,9 @@ const ShouldHandleZoom = ({ isEnabled, onToggle }) =>
       shouldHandleZoom prop which should be a function that returns a boolean.
     </p>
     <p>
-      Try it! Click this checkbox to enable zooming and uncheck it to disable
-      the zoom capabilities.
+      Try it! Click this checkbox to enable zooming{' '}
+      <strong>only when the image is clicked </strong>
```

Comment:
strange requirement for a space on the line above and before the closing strong tag here...

Reply:
This text doesn't break in the browser based on the line breaks here. This was needed to provide space between "clicked" and "and".